doc/tools/docgram/orderedGrammar

```diff
@@ -2292,6 +2292,11 @@ goal_selector: [
 | "[" ident "]"
 ]
 
+debug_univ_name: [
+| qualid
+| string
+]
+
```

Comment:
Move these lines below `range_selector` so it's outside of the range for `.. insertprodn goal_selector range_selector` in `ltac.rst`.  Otherwise doc_gram will insert it there.  Maybe you forgot to run `doc_gram_rsts` or didn't notice?

Reply:
Your refman job was failing with this message:

Warning, treated as error:
/builds/coq/coq/_build/default/doc/sphinx/proof-engine/**ltac.rst:425:Duplicate name ('token', 'debug_univ_name') (other is in /builds/coq/coq/_build/default/doc/sphinx/addendum/universe-polymorphism.rst)** attached to <desc_signature classes="['sig', 'sig-object']" first="False" ids="['grammar-token-debug_univ_name']" names="[('token', 'debug_univ_name')]"/>